README.rst

```diff
@@ -95,11 +98,13 @@ Map this data to a CDM build for the same deck (in this case deck 704: US Marine
 
 .. code-block:: python
 
-    name_of_model = "icoads_r3000_d704"
+    from cdm_reader_mapper.cdm_mapper import map_model
 
-    cdm_dict = cdm.map_model(
-        name_of_model,
+    name_of_model = "icoads_r300_d704"
+
+    cdm_dict = map_model(
         data_raw,
+        name_of_model,
```

Comment:
```suggestion
        imodel=name_of_model,
```

Reply:
It looks nicer somehow :blush:, isn't it?

Reply:
To take over this suggestion you can simply use the ``Commit suggestion`` button.